app/models/person.rb

```diff
@@ -203,6 +203,11 @@ def service_duration
 
   private
 
+  def check_zipcode
+    return unless zipcode.present? && !zipcode.match(/^(?:[1-9]|0(?!0{4}))\d{4}(?:[-\s]\d{4})?$/)
+    errors.add(:zipcode, "invalid format - must be 5 digits, ex. 12345")
```

Comment:
Should we change this error message to match the regex ? Zip and zip+4 `02134-1234` are both valid.